tests/data/config/valid_toml_numpydoc/test_package/numpydoc_style.py

```diff
@@ -0,0 +1 @@
+../../../../data/format/numpydoc/numpydoc_regression.py
```

Comment:
Could you just copy the contents of the file here instead of using a symlink? If we ever change the directory structure I can see myself wondering for 5 minutes why that would affect unrelated tests like this 😄

Reply:
Will do!